src/MercureBundle.php

```diff
@@ -31,13 +31,17 @@ public function build(ContainerBuilder $container): void
         $container->addCompilerPass(new class() implements CompilerPassInterface {
             public function process(ContainerBuilder $container): void
             {
+                if (!$container->hasDefinition(Authorization::class)) {
+                    return;
+                }
+
                 $definition = $container->getDefinition(Authorization::class);
                 if (
                     null === $definition->getArgument(1) &&
                     $container->hasParameter('session.storage.options')
                 ) {
                     $definition->setArgument(
-                        2,
+                        1,
```

Comment:
`Authorization`'s constructor has only 2 arguments, 0 and 1 from the container pov